tools/db_crashtest.py

```diff
@@ -293,8 +293,8 @@ def blackbox_crash_main(args, unknown_args):
         killtime = time.time() + cmd_params['interval']
 
         cmd = gen_cmd(dict(
-            cmd_params.items() +
-            {'db': dbname}.items()), unknown_args)
+            list(cmd_params.items()) +
```

Comment:
`gen_cmd({**cmd_params, 'db': dbname}, unknown_args)` is more pythonic in Py3, but this syntax is not available in Py2